LearningMissionLab/EducationDegree.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public enum EducationDegree
+    {
+        Unspecified,
+        Student,
+        Bachelor,
+        postgraduate,
```

Comment:
Rename postgraduate to PhD, move it below Master

Reply:
Fixed.